.github/actions/build/action.yml.j2

```diff
@@ -0,0 +1,94 @@
+name: build
+
+description: Build specified project
+
+inputs:
+  build-type:
+    required: true
+    type: string
+    description: One of ci / release
+  target-device:
+    required: true
+    type: string
+  platform:
+    required: true
+    type: string
+  use-container:
+    required: true
+    type: boolean
+  docker-image:
+    type: string
+    required: true
+  inter-repos-ro-access-token:
+    type: string
+    required: true
```

Comment:
What if you don't need access to any other repos?

Reply:
Removed.